### PR TITLE
add more logs to CI

### PR DESCRIPTION
**Pull request by eslutsky**

### CODE REVIEW

File: tests/suit/framework/framework.go [hunk 1]
```diff
@@ -137,7 +137,9 @@ func (f *Framework) BeforeEach() {
 		})
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 		f.Namespace = ns
-		f.AddNamespaceToDelete(ns)
+
+		//dont delete namespace after tests are done
```
Comment (ahadas) [inline tests/suit/framework/framework.go:141]: or to have a flag that is not set by default for deleting the generated namespace

File: .github/workflows/pull-request.yml [hunk 2]
```diff
@@ -77,15 +77,42 @@ jobs:
...
-            kubectl get pvc -A >> /tmp/artifacts/k8s-pvc.log
             kubectl cp konveyor-forklift/$(kubectl get po -n konveyor-forklift  -o=name | grep packstack|cut -d/ -f2):/var/log/cinder/volume.log /tmp/artifacts/k8s-packstack-cinder-volume.log
+            kubectl cp konveyor-forklift/$(kubectl get po -n konveyor-forklift  -o=name | grep packstack|cut -d/ -f2):/var/log/nova/nova-compute.log /tmp/artifacts/k8s-packstack-nova-compute.log
+            kubectl cp konveyor-forklift/$(kubectl get po -n konveyor-forklift  -o=name | grep packstack|cut -d/ -f2):/var/log/glance/api.log /tmp/artifacts/k8s-packstack-glance-api.log
+            kubectl get openstackvolumepopulator -A >> /tmp/artifacts/k8s-openstackvolumepopulator.log
+            kubectl logs -n konveyor-forklift $(kubectl get po -n konveyor-forklift  -o=name | grep openstack-populator) >> /tmp/artifacts/k8s-forklift-openstack-populator.log
+            kubectl logs -n konveyor-forklift $(kubectl get po -n konveyor-forklift  -o=name | grep forklift-volume-populator) >> /tmp/artifacts/k8s-forklift-volume-populator.log
```
Comment (ahadas) [inline .github/workflows/pull-request.yml:111]: it would be handy to get more complete logs (of all objects) from the generated namespace

Comment (eslutsky) [reply .github/workflows/pull-request.yml:111]: what do you mean ? there is no pods on the generated NS , only CRs.. we can have logs from pods only

File: tests/suit/framework/framework.go (hunk 1 above)
Comment (eslutsky) [reply tests/suit/framework/framework.go]: done, added flags `-delete-namespace` which can be added to the ginkgo execution parameter.

File: .github/workflows/pull-request.yml (hunk 2 above)
Comment (eslutsky) [reply .github/workflows/pull-request.yml:114]: done, see [commit](https://github.com/kubev2v/forklift/pull/212/commits/e022770f57d9b6cc130a22447ab7429b362d193f)

File: .github/workflows/pull-request.yml [hunk 3]
```diff
@@ -77,15 +77,45 @@ jobs:
...
```
Comment (ahadas) [inline .github/workflows/pull-request.yml:114]: we use the term populator pod for openstack/ovirt and populator-controller for the controller in the konveyor-forklift namespace so how about:
```suggestion
            kubectl logs -n konveyor-forklift $(kubectl get po -n konveyor-forklift  -o=name | grep forklift-volume-populator) >> /tmp/artifacts/k8s-forklift-volume-populator-controller.log
```

Comment (eslutsky) [reply .github/workflows/pull-request.yml]: ack and done

**Review by ahadas — APPROVED** [review]

**Pull request merged**